frontend/src/pages/learn/main-sources-of-retirement-income.tsx

```diff
@@ -20,30 +22,42 @@ const MainSourcesOfRetirementIncome: FC = () => {
 
   const learnMoreLinks = useMemo(
     () => [
+      {
+        href: '/learn/deciding-when-to-collect-public-pensions',
+        primary: t('learn-more.when-to-start.header'),
+        secondary: t('learn-more.when-to-start.description'),
+      },
       {
         href: '/learn/going-from-work-to-retirement',
         primary: t('learn-more.going-from-work-to-retirement.header'),
         secondary: t('learn-more.going-from-work-to-retirement.description'),
       },
       {
-        href: '/learn/canada-pension-plan-program',
-        primary: t('learn-more.canada-pension-plan-program.header'),
-        secondary: t('learn-more.canada-pension-plan-program.description'),
+        href: 'https://www.canada.ca/en/services/taxes/savings-and-pension-plans.html',
+        primary: t('learn-more.savings-and-pension.header'),
+        secondary: t('learn-more.savings-and-pension.description'),
       },
       {
-        href: '/learn/old-age-security-program',
-        primary: t('learn-more.old-age-security-program.header'),
-        secondary: t('learn-more.old-age-security-program.description'),
-      },
-      {
-        href: '/learn/main-sources-of-retirement-income',
-        primary: t('learn-more.sources-of-retirement-income.header'),
-        secondary: t('learn-more.sources-of-retirement-income.description'),
+        href: 'https://www.canada.ca/en/services/benefits/publicpensions/cpp/retirement-income-calculator.html',
+        primary: t('learn-more.income-calculator.header'),
+        secondary: t('learn-more.income-calculator.description'),
       },
     ],
     [t]
   )
 
+  const mobile = useMediaQuery(theme.breakpoints.down('sm'))
+
+  const Img = ({ pension, classes }: { pension: string; classes?: string }) => (
```

Comment:
Do not define components during render. React will see a new component type on every render and destroy the entire subtree’s DOM nodes and state. Instead, move this component definition out of the parent component “MainSourcesOfRetirementIncome” and pass data as props. `sonarlint(typescript:S6478)`

@Fbasham fixed this in other pages. Take a look with him.

Reply:
Alright, I went ahead and removed it.

Reply:
Alright, alt text fixed.